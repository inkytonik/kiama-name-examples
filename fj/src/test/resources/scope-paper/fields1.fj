// class/fields.spt:access field
class A extends Object {
  Object f;
  A(Object f) {
    super();
    this.f = f;
  }
}
new A(new Object()).f
