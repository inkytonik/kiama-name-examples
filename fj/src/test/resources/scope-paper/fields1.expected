src/test/resources/scope-paper/fields1.fj:3:10:info: f def: Object
  Object f;
         ^
src/test/resources/scope-paper/fields1.fj:4:12:info: f def: Object
  A(Object f) {
           ^
src/test/resources/scope-paper/fields1.fj:9:1:info: new A(new  ...: A
new A(new Object()).f
^
src/test/resources/scope-paper/fields1.fj:9:1:info: new A(new  ...: Object
new A(new Object()).f
^
src/test/resources/scope-paper/fields1.fj:9:7:info: new Object ...: Object
new A(new Object()).f
      ^
Object
